### PR TITLE
Swift5

### PR DESCRIPTION
**Pull request by Michael-AirMap**

### CODE REVIEW

File: AirMapSDK.podspec [hunk 1]
```diff
@@ -45,27 +45,34 @@ Pod::Spec.new do |s|
 		ui.platform = :ios
 		ui.frameworks = 'UIKit'
 		ui.dependency 'AirMapSDK/Core'
-		ui.dependency 'PhoneNumberKit', '< 3.0.0'
+		ui.dependency 'PhoneNumberKit'
```
Comment (adolfo) [inline AirMapSDK.podspec]: Let's pin a major version here with the optimistic operator. `~>`

File: AirMapSDK.podspec [hunk 2]
```diff
@@ -4,7 +4,7 @@ Pod::Spec.new do |s|
 	s.name              = 'AirMapSDK'
 	s.module_name       = 'AirMap'
 	s.author            = 'AirMap, Inc.'
-	s.version           = '3.0.0'
+	s.version           = '4.0.0.alpha.1'
```
Comment (adolfo) [inline AirMapSDK.podspec]: Let's go with 4.0.0.

File: AirMapSDK.podspec (hunk 1 above)
Comment (Michael-AirMap) [reply AirMapSDK.podspec]: Update to 'PhoneNumberKit', '~> 3.0'

File: AirMapSDK.podspec (hunk 2 above)
Comment (Michael-AirMap) [reply AirMapSDK.podspec]: Done

**Pull request merged**